rpc/manager/manager.proto

```diff
@@ -203,6 +203,13 @@ message InterceptInfo {
   string pod_ip = 10;
   int32 sftp_port = 11;
 
+  // The directory where the client mounts the remote mount_point. Only
+  // set when obtaining InterceptInfo from the user daemon.
+  string client_mount_point = 2;
```

Comment:
The only significance of the numbers is to send data in the correct order so that compatibility is maintained. The "procedure" is to ensure that we never introduce a compatibility issue. In this case, the number 2 was reserved a very long time ago when we first introduced connection tunnels between the client and the TM (around the release of 2.3.0) and we don't support that old clients anyway.

IMO, it's important to group attributes together contextually. The numbers are mostly irrelevant as long as we notice them when we do modifications. Contextually, I felt that the `client_mount_point` should be close to `pod_id`, `sftp_port` and `mount_point` since those three four are all about the remote mount, so I'd like to keep it that way even though the number sequence is 10, 11, 2, 16.